### PR TITLE
Don't upload small tensors as uniforms to packed op programs

### PR DESCRIPTION
**Pull request by astojilj**
Trivial: modify the check to prevent upload to uniform when
texData.isPacked == true and program.usesPackedTextures == true.

BUG

#### Description
I got this reproduced from batchnorm tests with [packed binary PR](https://github.com/tensorflow/tfjs-core/pull/1423) on and [this modification of batchnormalization4D tests](https://gist.github.com/astojilj/15b2e02089e733f5512463aeff90a2e4).
This test would run existing batchnormTests with WEBGL_PACK flag on.
although texData.isPacked == true, and program.usesPackedTextures == true. the logic would still upload the tensor to uniform.

---

##### For repository owners only:

Please remember to apply all applicable tags to your pull request.
Tags: FEATURE, BREAKING, BUG, PERF, DEV, DOC, SECURITY

For more info see: https://github.com/tensorflow/tfjs/blob/master/DEVELOPMENT.md

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-core/1429)

### CODE REVIEW

**Comment by astojilj** [comment]
@annxingyuan PTAL. 
I'm aware that you have a [patch in queue](https://github.com/tensorflow/tfjs/issues/821) that enables sampling from uniforms.

**Comment by astojilj** [comment]
> LGTM - thanks a lot @astojilj for catching this! Feel free to merge.

Thanks. I don't have write access required to merge it.
